lib/features/children/add_member/widgets/add_member_form.dart

```diff
@@ -34,22 +35,69 @@ class _AddMemberFormState extends State<AddMemberForm> {
   int _allowanceController = 15;
   final formKeyChild = GlobalKey<FormState>();
   final formKeyParent = GlobalKey<FormState>();
+  Timer? _timer;
+  Duration _heldDuration = Duration.zero;
+
+  void _startTimer(void Function() callback) {
+    _timer = Timer.periodic(Duration(milliseconds: 240), (_) {
+      _heldDuration += Duration(milliseconds: 240);
+      callback();
+    });
+  }
+
+  void _stopTimer() {
+    _timer?.cancel();
+    _heldDuration = Duration.zero;
+  }
+
+  @override
+  void dispose() {
+    _stopTimer();
+    super.dispose();
+  }
 
   void _incrementCounter() {
-    if (_allowanceController > 998) {
+    final isHeldDurationShortEnoughForIncrement1 =
+        (_heldDuration.inMilliseconds > 1000) && _allowanceController > 994;
```

Comment:
Maybe it would be better to put all the numbers into constants / variables. 
Its really easy to make mistake right now if you wanna change one of the values.

Also the 994, 984, 998 feels like it should be a calculation instead of hard values. (if I understand it correctly)